probability.ipynb

```diff
@@ -1485,6 +1583,240 @@
    "source": [
     "gibbs_ask('Cloudy', dict(Rain=True), sprinkler, 200).show_approx()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inference in Temporal Models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before we start, it will be helpful to understand the structure of a temporal model. We will use the example of the book with the guard and the umbrella. In this example, the state $\\textbf{X}$ is whether it is a rainy day (`X = True`) or not (`X = False`) at Day $\\textbf{t}$. In the sensor or observation model, the observation or evidence $\\textbf{U}$ is whether the professor holds an umbrella (`U = True`) or not (`U = False`) on **Day** $\\textbf{t}$. Based on that, the transition model is \n",
```

Comment:
I suggest instead of using latex to bold/italicize text to use the Markdown syntax. For italics, you wrap the phrase in single asterisks and for bold you wrap it in double asterisks. We have had some problems with Github and Latex before, so we are not taking any chances.

If possible, I would like you to convert Latex to Markdown. It's not *that* important, but I would prefer it this way just to be safe.